pages/agent/v3/elastic_ci_stack_for_ec2_mac/autoscaling_mac_metal.md

```diff
@@ -69,9 +69,9 @@ can access the instance.
 	- Grow the AFPS container to use all the available space in your EBS root disk if needed, see the [AWS user guide](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-mac-instances.html#mac-instance-increase-volume)
 1. Using a VNC session (run SSH port forwarding `ssh -L 5900:localhost:5900 ec2-user@<ip-address>` if direct access is not available):
 	1. Sign in as the `ec2-user`
-	1. Enable *Automatic login* for the `ec2-user` in *System Preferences* > *Users & Accounts* > *Login Options*
-	1. Disable *Require password* in *System Preferences* > *Security & Privacy* > *General*
-	1. Disable the screen saver in *System Preferences* > *Desktop & Screen Saver* > *Show screen saver after*
+	1. Set *Automatically log in as* to `ec2-user` in *System Preferences* > *Users & Groups*
+	1. Set an empty password in *System Preferences* > *Login Password*
+	1. Set *Start Screen Saver when inactive* to `Never` in *System Preferences* > *Lock Screen*
```

Comment:
Thanks for your PR @steveh . As we are changing the formatting for UI references in the docs (https://github.com/buildkite/docs/pull/2739), I'm going to make these suggested changes...

```suggestion
	1. Set **Automatically log in as** to `ec2-user` in **System Preferences** > **Users & Groups**
	1. Set an empty password in **System Preferences** > **Login Password**
	1. Set **Start Screen Saver when inactive** to `Never` in **System Preferences** > **Lock Screen**
```

As part of PR 2739, I'm going to update our [Markdown syntax guidelines for UI references](https://github.com/buildkite/docs/blob/main/styleguides/markdown-syntax-style.md#ui-elements). On that note, if you did want to use italics in the docs (going forward, this should only be for key terms), then please use underscores (`_`) only.